src/test/java/io/smartcat/migration/migrations/data/PopulateItemByNumberAndExternalIdMigration.java

```diff
@@ -0,0 +1,36 @@
+package io.smartcat.migration.migrations.data;
+
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.querybuilder.QueryBuilder;
+import io.smartcat.migration.DataMigration;
+import io.smartcat.migration.exceptions.MigrationException;
+
+import java.util.List;
+
+public class PopulateItemByNumberAndExternalIdMigration extends DataMigration {
+
+    public PopulateItemByNumberAndExternalIdMigration(final int version) {
+        super(version);
+    }
+
+    @Override
+    public String getDescription() {
+        return "Create entry for each item in items by id table into new items by number and external id table";
+    }
+
+    @Override
+    public void execute() throws MigrationException {
+        try {
+            final PreparedStatement preparedStatement =
+                    session.prepare("INSERT INTO items_by_number_external_id (id, number, external_id) VALUES (?, ?, ?);");
+
+            final List<Row> rows = session.execute(QueryBuilder.select().from("items_by_id")).all();
+            for (Row row : rows) {
```

Comment:
Maybe add fetch size to this, not that we have this many items but it shows that we know how it works and good practice.